src/FamilyHubs.ReferralUi.Ui/FamilyHubs.ReferralUi.Ui.csproj

```diff
@@ -27,4 +28,15 @@
     <PackageReference Include="Serilog.Sinks.Console" Version="4.1.0" />
     <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.26.0" />
   </ItemGroup>
+
+	<ItemGroup>
+		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
```

Comment:
It's preferable to only test the public interface (but testing non-public can sometimes be useful).

Reply:
I have made the class used by redis Cache internal and obfuscated it to make life a little more difficult for any potential hacker. We might decide to do more at a latter stage